### PR TITLE
React capstone project:( Weather ) 

### PR DESCRIPTION
**Pull request by AliShahSafdari**
In this PR add features:

- Use Air pollution API.
- Add Home page.
- Add Details page.
- Add unit tests for pure functions in my Redux code.
- Use Jest for unit testing.
- live link(https://weather-1glk.onrender.com).
- video link(https://loom.com/share/bc89172a729b43439011e75df902b984?sid=3e593e3e-32e5-404a-9996-50c81494d646)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,13 +69,13 @@
...
 
-- [Live Demo Link](comin soon)
+- [Live Demo Link](https://weather-1glk.onrender.com/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
 ## 🎥 Video Link <a name="live-demo"></a>
--[Video describing the project](comin soon)
+-[Video describing the project](https://loom.com/share/b916f62ae3fd458d897f48306b386391?sid=b6d96a85-42a2-40bf-bd5c-445a99c5f3c7)
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Good presentation 👏🏿 Your presentation should also contain a highlight about a piece of the code you find  interesting or you are  proud of making based on the [project documentation](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation) requirements, please record another video following all the requirements 🤗

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @AliShahSafdari 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good file structure  
✔️  Design looks like the Behance design
✔️Tests written 
✔️ Home page has a dropdown to  filter by categories 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AliShahSafdari) [reply README.md]: Thanks, I add a new video.

**Review by laiifuu — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: 

## Hi @AliShahSafdari :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

friends-joey

## Highlights
- [x] Changes required were implemented :heavy_check_mark: 
- [x] Application works as expected :heavy_check_mark: 
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**